packages/api-browser/window.js

```diff
@@ -2,10 +2,20 @@ if (!window.ssf) {
   window.ssf = {};
 }
 
+window.accessableWindows = [];
```

Comment:
access*a*ble => access*i*ble

If names are unique, then I think an object might be better here - mapping name to window object. Accessing should be easier and faster, as it would just be `accessibleWindows[name] ` (not `accessibleWindows.find(...)`) and we won't need to create a new object for each window.

However, if order is important and the array is needed, then I don't think the `id` would be needed on the new object pushed into the array, as it's already on the `name` property of the window - could just push the window directly into the array?

If possible, I think we should avoid polluting the global namespace like this. Perhaps, for now, `accessibleWindows` could be a `const` and there could be `static` methods on `Window` to allow this to be get/set?